File: solutions/typescript/darts/1/darts.ts
export function score(x: number, y: number): number {
  const coordinate = Math.sqrt(x*x + y*y);

  if (coordinate <= 1) return 10;
  if (coordinate <= 5) return 5;
  if (coordinate <= 10) return 1;
  return 0;
}

File: solutions/python/darts/1/darts.py
import math 

def score(x, y):
    x = abs(x)
    y = abs(y)
    c = math.sqrt(x**2 + y**2)

    if c <= 10:
        if c <= 5:
            if c <= 1:
                return 10
            return 5
        return 1
    else: 
        return 0